src/video-grid/VideoTileContainer.tsx

```diff
@@ -66,6 +66,7 @@ export function VideoTileContainer({
     speaking,
     stream,
     purpose,
+    debugInfo,
```

Comment:
This should be added to the props interface

Reply:
Props interface? This is coming from a hook

Reply:
No worries